public/components/incontext_insight/index.scss

```diff
@@ -107,10 +107,75 @@
 }
 
 .incontextInsightPopoverBody {
-  max-width: 400px;
+  max-width: 486px;
+  min-width: 486px;
   width: 100%;
 }
 
+.incontextInsightGeneratePopoverTitle {
+  // TODO: Remove this one paddingSize is fixed
+  padding: 0 !important;
+  margin-bottom: 0 !important;
+  min-height: 30px;
+  max-height: 30px;
+  border: none;
+
+  :first-child {
+    border: none;
+  }
+
+  .euiBadge__text {
```

Comment:
Why we need to override oui styles?

Reply:
To match UX design.

Reply:
Replaced CSS styles with component property